src/components/button/templates/component/template.js

```diff
@@ -108,8 +126,15 @@ export function componentTemplate({ props } : { props : Object }) : string {
 
     let labelTag = '';
 
+    // tag content below button allowed only for checkout, branded buynow, credit button;
+    // also when style.fundingicons is false
     if (!props.style.fundingicons) {
-        labelTag = conf.tagline && content[`${label}_tag`] ? content[`${label}_tag`] : '';
+        labelTag = conf.tagline && content[`${conf.tagcontent}`] ? content[`${conf.tagcontent}`] : '';
+    }
+
+    // Hack for buy now branded option to support custom css
```

Comment:
Not sure if my last comment got lost, but just making a note here -- let's use a different class name for branded / unbranded